packages/mobile/src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -235,8 +235,9 @@ function FiatExchangeAmount({ route }: Props) {
     <SafeAreaView style={styles.container} edges={['bottom']}>
       <Dialog
         isVisible={showingInvalidAmountDialog}
-        actionText={t('invalidAmountDialog.dismiss')}
-        actionPress={closeInvalidAmountDialog}
+        secondaryActionText={t('invalidAmountDialog.dismiss')}
+        secondaryActionPress={closeInvalidAmountDialog}
```

Comment:
nit: I think it would be better to have a way to say whether the primary action is highlighted or not.
Feels a bit weird to pass a secondary action without a primary action.

Reply:
I will do those changes, and see how the code looks like.
However, I think it's ok to avoid forcing having at least an action. So, we can have dialogs without actions, that can be dismissible by pressing in the background.

Reply:
@jeanregisser I did these changes. Let me know if you feel it's better now.